src/sagas/contact.js

```diff
@@ -0,0 +1,22 @@
+import { takeEvery } from 'redux-saga/effects';
+
+import {
+    GET_CONTACT,
+} from '../constants';
+
+import { updateCurrentContact } from 'actions';
+
+function* getContact({ uid }) {
+    const contactRef = `/contacts/${uid}`
+    window._FIREBASE_DB_.ref(contactRef)
```

Comment:
Thoughts on `window._FIREBASE_DB_.ref( '/contacts/' + ${uid} )`?

Or, keep it as is and ensure that `uid` exists first?

Reply:
how about
```javascript
window._FIREBASE_DB_.ref( `/contacts/${uid || ''}`)
```

or would we want to throw an error if there's no uid specified?